awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupForm/InventoryGroupForm.test.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mountWithContexts } from '@testUtils/enzymeHelpers';
+import InventoryGroupForm from './InventoryGroupForm';
+
+const group = {
+  id: 1,
+  name: 'Foo',
+  description: 'Bar',
+  variables: 'ying: false',
+};
+describe('<InventoryGroupForm />', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mountWithContexts(
+      <InventoryGroupForm
+        handleSubmit={jest.fn()}
+        handleCancel={jest.fn()}
+        group={group}
+      />
+    );
+  });
+  afterEach(() => {
+    wrapper.unmount();
+  });
+  test('initially renders successfully', () => {
+    expect(wrapper.length).toBe(1);
```

Comment:
I think it's good that we that it mounts once, but the following tests probably don't need to.